### DISCUSSION TITLE
No module named 'faiss'

### DISCUSSION THREAD

**Discussion by SinghShreyas** [castorini/pyserini#1232] [Q&A] [answered]
Hello,

Whenever I am trying to do this import `from pyserini.search import JLuceneSearcherResult`. I am getting ModuleNotFoundError saying no module named 'faiss'.

Any help with this will be appreciated.

Thanks!
![Screenshot 2022-07-12 at 12 34 42 am](https://user-images.githubusercontent.com/25030681/178289180-8c096460-9d02-40ba-a59d-cb6e5e950902.png)

**Comment by lintool** [comment, marked as answer]
This is covered in our installation instructions - please see this: https://github.com/castorini/pyserini/#installation

**Comment by SinghShreyas** [comment]
@lintool  Thanks for the quick reply. It really helped and I was able to fix the problem. My bad that I missed the detailed instruction. 
Thanks again!
